articles/azure/azs-how-create-vm-ps.md

```diff
@@ -21,7 +21,7 @@ toc_mdlink: azs-how-create-vm-ps.md
 ## Overview
 
 With UKCloud for Microsoft Azure, you can leverage the power of Microsoft Azure to create virtual machines (VMs) for your on-premises applications.
-As UKCloud for Microsoft Azure is built on UKCloud’s assured, UK-sovereign multi-cloud platform, those applications can work alongside other cloud platforms, such as Oracle,
+As UKCloud for Microsoft Azure is built on UKCloud's assured, UK-sovereign multi-cloud platform, those applications can work alongside other cloud platforms, such as Oracle,

```

Comment:
This was automatically updated when I saved the file in VS.  As it stands, I'm unable to find a way of reverting this particular change.

Reply:
Do not think this would be a problem but we need to render it locally to test - we can go over this tomorrow.

Reply:
Have rendered this doc locally and all seems fine.